lostfiles.py

```diff
@@ -355,5 +355,9 @@ def collect_tracked_files() -> Set[str]:
         raise AssertionError("No tracked files found. This is probably a bug!")
     return files
 
+def check_requirements() -> None:
```

Comment:
@inode64 Please remove that function, it should not be required to have it installed via portage. I can see that this can happen during development, but the correct way is to create a [virtualenv](https://docs.python.org/3/library/venv.html) and install it inside of it using the `setup.py` file, e.g.:

```
$ python -m venv venv
$ source venv/bin/activate
(venv) $ python setup.py install
(venv) $ ./lostfiles
```

Reply:
Done,

Reply:
But since I had not seen it in the installation documentation, I thought that it could only be done through an emerge